_includes/yellow/yellow-install-haos-cm4.md

```diff
@@ -1,110 +1,156 @@
+- Wait for Home Assistant Yellow to start booting from the USB flash drive.
+- During boot, the Yellow LED will blink steadily.
+- Once the installer is ready, the Yellow LED will be constantly on for 5 seconds.
+- **Troubleshooting**: If the LED is stuck in steady blinking or constantly on for an extended period of time, this could indicate that the installer cannot connect to the internet. Follow this procedure to troubleshoot.
 
-   ![Clip showing the LED pattern with the yellow LED blinking steady](/static/img/yellow/led-pattern-blink-steady.webp)
+{% endstepContent %}
+{% endstep %}
 
-1. Waiting.
-   - During installation, the LED will blink fast.
-   - Wait for Home Assistant Yellow to finish installing the Home Assistant Operating System.
-   - Once the installer finishes installing the Home Assistant Operating System, only the red LED will be on.
+{% step "Waiting" %}
+{% image "/static/img/yellow/led-pattern-install-blink-fast.webp" "Clip showing the LED pattern with the yellow LED blinking fast" %}
+{% stepContent %}
 
-   ![Clip showing the LED pattern with the yellow LED blinking fast](/static/img/yellow/led-pattern-install-blink-fast.webp)
+- During installation, the LED will blink fast.
+- Wait for Home Assistant Yellow to finish installing the Home Assistant Operating System.
+- Once the installer finishes installing the Home Assistant Operating System, only the red LED will be on.
 
-1. Disconnecting everything.
+{% endstepContent %}
+{% endstep %}
 
-   - Remove the USB flash drive.
-   - Remove the power cable.
+{% step "Disconnecting everything" %}
+{% image "/static/img/yellow/plug-nothing.jpeg" "Image showing the connectors of the Yellow with nothing connected to them" %}
+{% stepContent %}
 
-   ![Image showing the connectors of the Yellow with nothing connected to them](/static/img/yellow/plug-nothing.jpeg)
+- Remove the USB flash drive.
+- Remove the power cable.
 
-1. Reconnecting power.
+{% endstepContent %}
+{% endstep %}
 
-   - Reinsert the Ethernet cable.
-   - Reinsert the power cable to start Home Assistant (if PoE is not used).
+{% step "Reconnecting power" %}
+{% image "/static/img/yellow/plug-ethernet-power.jpeg" "Image showing the the Yellow with Ethernet and power connected" %}
+{% stepContent %}
 
-   ![Image showing the the Yellow with Ethernet and power connected](/static/img/yellow/plug-ethernet-power.jpeg)
+- Reinsert the Ethernet cable.
+- Reinsert the power cable to start Home Assistant (if PoE is not used).
 
-1. Get started with Home Assistant.
+{% endstepContent %}
+{% endstep %}
 
-   1. Initial startup may take a while, depending on your internet connection.
-      ![Home Assistant user interface on a screen](/static/img/green/getting_started_04.png)
-   1. Install the app.
+{% step "Get started with Home Assistant" %}
```

Comment:
This is a long step, containing 2 actions:
 - Scanning and installing app (in the current layout, the QR codes are small, and easy to scan wrong)
 - Accessing home assistant in that ap

Reply:
thx. topic for a separate PR.